src/library/components/Button.tsx

```diff
@@ -48,10 +61,14 @@ const Button = forwardRef(
     }
 
     return (
-      <button ref={ref} className={`${classes} ${className || ''}`} {...buttonAttrs}>
-        {!!(icon && iconAlignment === 'left') && <Icon name={icon} size={iconSize} className="bsds-icon-left" />}
+      <button ref={ref} className={`${classes} ${className || ''} ${buttonSizeClass}`} {...buttonAttrs}>
+        {!!(icon && iconAlignment === 'left') && (
```

Comment:
Would this work without the `!!` (keeping the parens)?

Reply:
We'll have to keep boolean coercion due to eslint rules.